superbench/common/utils/network.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Network Utility."""
+
+import socket
+import subprocess
+from contextlib import closing
+
+
+def get_free_port():
+    """Get a free port in current system.
+
+    Return:
+        port (int): a free port in current system.
+    """
+    with closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as s:
```

Comment:
What's the benefit of using `with closing()`?

Reply:
it's an error-handling for socket exception